decidim-admin/spec/system/admin_manages_organization_areas_spec.rb

```diff
@@ -75,16 +75,36 @@
       end
 
       it "can delete them" do
-        within find("tr", text: translated(area.name)) do
-          accept_confirm { click_link "Delete" }
-        end
+        click_delete_area
 
         expect(page).to have_admin_callout("successfully")
 
         within ".card-section" do
           expect(page).to have_no_content(translated(area.name))
         end
       end
+
+      context "when a participatory space associated to a given area exist" do
+        let!(:process) { create(:participatory_process, organization: area.organization, area: area) }
+
+        it "can not be deleted" do
+          click_delete_area
+          expect(area.reload.destroyed?).to be false
+          expect(page).to have_admin_callout("Area has depedent spaces")
+        end
+      end
+    end
+  end
+
+  #---------------------------------------------------
+
+  private
+
+  #---------------------------------------------------
```

Comment:
Please remove this.

Reply:
Are we a community or we aren't? because we all have different styles when coding. We already have rubocop and the rest of the linters to keep the cohesion of the code.
This comments are compatible with ALL linters. I shoud remove them only because you don't like'em?

Reply:
Rubocop is used to check the styles of the Ruby code, the code that's actually executed. We don't have any checks for comments. We could discuss unifying all the method documentation, for example, into a single, unified format. That would enable generating nice docs, for example.

Since there's a lot of developers involved in the project, we need to enforce some kind of coherence in the coding style. This addition here does not comply the general coding style of Decidim, whether it's implicit or explicitly defined. Apart from that, it's not an industry standard in any project I've been to, nor in the Ruby open-source community. 

Also, this is an already discussed issue, which I sincerely considered closed. I see now that for you it wasn’t. In the previous discussion external developers jumped in and it was agreed that it was better not to have this kind of comments/code breakers. Since it’s a pattern that it’s not widely used by Decidim contributors it doesn’t seem necessary to reopen this debate.

Whether I like this pattern or not is irrelevant. Decidim has a coding style, and part of my job is ensuring this keeps being applied. Please, I ask you to reconsider and undo this change in #5050.

Thanks!